src/internal/Popover.js

```diff
@@ -372,12 +372,13 @@ class Popover extends Component<DefaultProps, AllProps, void> {
       onExiting,
       onExited,
       elevation,
+      autoFocus,
       ...other
     } = this.props;
 
     // FIXME: props API consistency problem? - `...other` not spread over the root
```

Comment:
I wasn't sure this was a good idea because of the number of props passed in compound to the different components in this render function.